worker-markup-core/pom.xml

```diff
@@ -41,7 +41,18 @@
             <groupId>com.github.cafdataprocessing</groupId>
             <artifactId>util-email-content-segregation</artifactId>
             <version>2.0.0-5</version>
-        </dependency> 
+        </dependency>
+        <dependency>
+            <groupId>com.github.cafdataprocessing</groupId>
+            <artifactId>worker-document-interface</artifactId>
+            <version>${caf.worker-document.version}</version>
```

Comment:
These two versions should be coming in through dependency management rather than being specified as a property.